src/land/imsfv3_scf2ioda.py

```diff
@@ -63,24 +69,26 @@ def _read(self):
             if iodavar == 'snowCoverFraction':
                 self.varAttrs[iodavar, iconv.OvalName()]['units'] = '1'
                 self.varAttrs[iodavar, iconv.OerrName()]['units'] = '1'
+                # should use float_missing_value
                 self.varAttrs[iodavar, iconv.OvalName()]['_FillValue'] = -999.
                 self.varAttrs[iodavar, iconv.OerrName()]['_FillValue'] = -999.
```

Comment:
```suggestion
                # should use float_missing_value
                self.varAttrs[iodavar, iconv.OvalName()]['_FillValue'] = float_missing_value
                self.varAttrs[iodavar, iconv.OerrName()]['_FillValue'] = float_missing_value
```

@YoulongXia-NOAA here is the code looking explictly for the `-999.` value or can I change it to `float_missing_value`... of course also need to modify the code to do the same change

Reply:
@BenjaminRuston, I am not sure for that case. The land-ims-proc code is developed by Clara Draper and she also use it to do snow da assimilation. She produced the data example (C48) to me and then I develop this ioda converter. I do not think that we need to change code to replace -999. values. If you want to change -999., you can use if variable <0 or =-999., and replace it into the value you want. But we need to double check with Clara and see if this is OKAY for her assimilation code.